### ISSUE TITLE
Namespaces are defined multiple times

### ISSUE THREAD

**Issue by berezovskyi** [oslc-op/oslc-specs#91]
I see no rationale for that. They are defined in every part of the spec and both under the RDF Namespaces heading under the front matter as well as the section §1.3 1.3 Typographical Conventions and Use of RFC Terms.

First spotted in the QM spec.

**Comment by berezovskyi** [comment]
I think it is best the normative content is defined once in each spec.

**Comment by jamsden** [comment]
In the CM 3.0 spec, I see the oslc_cm namespace defined once in section 2.3 Namespaces, along with the namespace referenced in the document. The RDF Namespaces: element in the document summary section is only for convenient reference and was something I think carried over from the W3C ReSpec template.

**Comment by berezovskyi** [comment]
What I was referring to is that the aforementioned section is duplicated in multiple parts (!) of the same spec, which I consider to be unnecessary and potentially hazardous as normative content is defined in two places and may diverge at some point in time.

**Comment by ndjc** [comment]
The section mentioned by Andrew is labeled 'conventions', and is normative, and both defines the qm namespace and references the other namespaces used throughout the spec. This section appears in both quality-management-spec.html and in quality-management-vocab.html.

I tend to agree that this information need only appear once in the spec.

I think the vocabulary document is a good place to define the QM namespace - but one could argue that the main spec is a good place to list the non-normative references to other namespaces used.

The block in the header showing namespaces is indeed originally from the W3C spec format. We could omit that, or we could use that as the non-normative reference to namespaces, removing the need to list them a second time in the 'conventions' section.

So my suggestion is this:

1. Add all defined and referenced namespaces to the 'namespaces' block in the conf section for each document (each part), to be rendered as a block entitled 'Namespaces used in this document'.
1. Remove all instances of namespace definitions from the 'conventions' section - they are not normative (since a namespace prefix is entirely a local typographic convenience and has no normative existence or meaning)
1. Remove all references to other namespaces from the 'conventions' section, since they duplicate what we have added to the document header. For many specs, this will leave the conventions section empty, making the spec smaller and simpler.

**Comment by berezovskyi** [comment]
@jamsden if you agree with the resolution, we can start making the changes.

**Comment by jamsden** [comment]
Sure - but we need to think about what needs to be done vs. what would be nice to have re: these continual changes. We're way behind where we should be on getting the OSLC specs to OASIS Standards. And moving to an OP has set us back even further in getting query and TRS addressed. 

Perhaps we can use some severity/priority indicator and establish a backlog that captures these nice to haves and distinguish them from things we need to do before we can publish.

**Comment by berezovskyi** [comment]
@jamsden I have copied Priority labels we have in Lyo, I have set this issue to have a Medium priority.

I have also set up a project board, feel free to order the backlog items as you see fit: https://github.com/oslc-op/oslc-specs/projects/2?fullscreen=true

**Comment by jamsden** [comment]
I am not going to address this issue in Core or CM PS01 since it will have little or no impact on the actual rendered HTML.
